src/app/Fake.Deploy/DeploymentHelper.fs

```diff
@@ -29,7 +29,7 @@ let getActiveReleases dir = !!(dir @@ deploymentRootDir @@ "**/active/*.nupkg")
 
 /// Retrieves the NuSpec information for the active release of the given app.
 let getActiveReleaseFor dir (app : string) = 
-    !!(dir @@ deploymentRootDir @@ app @@ "/active/*.nupkg")
+    !!(dir @@ deploymentRootDir @@ app @@ "active/*.nupkg")
```

Comment:
I thought I saw one in a previous issue thread, but I couldn't find it either.  From when I was first searching for why @@ behaved the way it did.  

I was thinking @@+ as it implies its like the old @@ but with more, but its ugly.  Or something more like ++ but that smells of plain concatenation.  <+> smells of haskell monad combinators but isn't bad.  :: isn't overrideable.  Nor is anything \ related.  |+ might work.  In any event, ive got it set to @@+ in my code, but if you come up with something better, let me know and I'll swap it out. I'm still working on a unit test that takes into consideration the different behaviors of Path.Combine under different platforms.